sfmc_release_notes/SF_managed_clusters_ga_release_notes.md

```diff
@@ -0,0 +1,40 @@
+# Microsoft Azure Service Fabric managed clusters general availability release
+
+Azure Service Fabric managed clusters are now generally available. We first introduced Service Fabric managed clusters in October, 2020. Service Fabric managed clusters enable customers to deploy Service Fabric clusters as a single ARM resource, and reduce the complexity associated with common cluster operations such as scaling, and certificate rollovers. In this release we are bringing additional capabilities such as availability zone spanning, stateless node types, application secrets, application managed identities, and the ability to configure cluster upgrades.
+
+To utilize these new features, you must update the API version in the ARM template to *2021-05-01*.
+
+In this release, Service Fabric managed clusters are now available in all Azure public cloud regions and will roll out to Azure special clouds in the coming weeks.
+ 
+The following APIs are updated as part of this release:
+
+| API |
+|---------|
+| Microsoft.ServiceFabric/managedclusters |
+| Microsoft.ServiceFabric/managedclusters/nodetypes |
+| Microsoft.ServiceFabric/managedclusters/applications |
+| Microsoft.ServiceFabric/managedclusters/applicationTypes |
+| Microsoft.ServiceFabric/managedclusters/applications/services |
+
+## Features
+
+* [Availability zone spanning](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-availability-zones) - You can now deploy Service Fabric managed clusters across availability zones to increase resiliency against zone failures.
```

Comment:
Do we want to align this with the marketing speak?
You can now deploy Service Fabric managed clusters across availability zones to increase resiliency against zone failures.

You can now deploy Service Fabric managed clusters across availability zones to eliminate single points of failure. (going off memory, I think its on our doc page)